### PR TITLE
Mobile labs brexit banner

### PR DESCRIPTION
**Pull request by NathanielBennett**
## What does this change?

The pesky dudes at mobile labs in New York City want another banner upsell, like the one here: https://github.com/guardian/frontend/pull/13201 (follow the link to see a screenshot - this one looks exactly the same except with different text )
## Request for comment

@gtrufitt

### CODE REVIEW

File: common/app/conf/switches/FeatureSwitches.scala [hunk 1]
```diff
@@ -441,4 +441,15 @@ trait FeatureSwitches {
...
   )
+
+  //Owener Dotcom participation
+  val MobileLabBrexitBanner = Switch(
+    SwitchGroup.Feature,
+    "mobile-lab-brexit-banner",
+    "When ON, we present a promo banner to android, in the the UK pollitics or EU referendum sections",
+    owners = Seq(Owner.withGithub("NathanielBennett")),
+    safeState = Off,
+    sellByDate = new LocalDate(2016, 6, 24),
```
Comment (gtrufitt) [inline common/app/conf/switches/FeatureSwitches.scala]: Sunday ;)

Comment (gtrufitt) [inline common/app/conf/switches/FeatureSwitches.scala]: oh no nevermind, it's June :D

Comment (gtrufitt) [inline common/app/conf/switches/FeatureSwitches.scala]: Are you here Friday? Might be better to put it mid-week next week to give you more time to change it, then they can just switch it off when it is done.

Comment (NathanielBennett) [inline common/app/conf/switches/FeatureSwitches.scala]: Will anyone be here on Friday?  
Good point. Thanks

**Comment by gtrufitt** [comment]
👍

File: common/app/conf/switches/FeatureSwitches.scala (hunk 1 above)
Comment (NathanielBennett) [inline common/app/conf/switches/FeatureSwitches.scala]: Are you keeping tabs on my holidays?

**Pull request merged**